### DISCUSSION TITLE
HELP Please

### DISCUSSION THREAD

**Discussion by Ad-Astra-R** [AnubisNekhet/AnuPpuccin#116] [Snippets]
Hello 

I am a totaly newbie and by the way I'm French.... So I will do my best !

I have download this Theme because I find it very Gorgeous but how are you supposed to configure it ? I even don't understand where informations are.... 

So please, Is there some one who can help me... ? 
Please <3 <3 <3

**Comment by ms3056** [comment]
First - install this theme from inside of Obsidian. Second, install Style Settings. third - configure in Style Settings. 

If you want the snippets download them from GitHub and put them in your snippets directory. Enable them and configure inside of the Style Settings plugin.

**Comment by jerrysoft-x** [comment]
I've installed AnuPpuccin and Style Settings, and then what to do?
I need some more instructions.
And no idea where to find the snippets.
Hope someone can guide me, thanks in advance!

**Reply by syrth-io** [reply]
there is a folder on the main page of the repository named "[snippets](https://github.com/AnubisNekhet/AnuPpuccin/tree/main/snippets)". click on the snippet you want, and then you can download it directly into your vault by clicking on "RAW", then CTRL-S or right-click and select "Save Page As..." (on windows--not sure about mac, sorry!) and navigate to the .obsidian/snippets folder of your vault, and save the CSS file there.

then you need to toggle the snippet on either via the Obsidian settings in the Appearance tab, or via a plugin such as MySnippets which lets you toggle snippets on and off by a pop-up menu activated in the status bar.

a lot of this is more knowledge of Obsidian found by reading the official vault and exploring the forum, plugins, videos, etc and not so much the theme itself, so you can reapply these steps to any snippet regardless of theme-dependence or independence! (:
